packages/storybook/src/theme.ts

```diff
@@ -33,6 +33,7 @@ export const theme = {
       fontFamily: 'heading',
       lineHeight: 'heading',
       fontWeight: 'heading',
+      fontSize: 32,
```

Comment:
Still getting used to this but think we can set this to 5 to reference the [5]th fontsize.

Reply:
Ah, ya we can. I more did this to show how to overwrite with your own values, but you're right that it's probably better to use `fontSize: 5`